### PR TITLE
ET-298 Merge values from duplicate filter sections in Insights

### PR DESCRIPTION
**Pull request by snowcar**

### CODE REVIEW

File: edx/analytics/tasks/student_properties_per_ora_tags_dist.py [hunk 1]
```diff
@@ -90,11 +90,12 @@ def mapper(self, line):
         student_properties = event.get('context').get('asides', {}).get('student_properties_aside', {})\
             .get('student_properties', {})
 
-        overload_items = {'course': course, 'term': run}
+        overload_items = {'course': course, 'term': run, 'courses': course}
```
Comment (dpowell) [inline edx/analytics/tasks/student_properties_per_ora_tags_dist.py]: Do we need the same for terms, then?

Comment (snowcar) [reply edx/analytics/tasks/student_properties_per_ora_tags_dist.py]: actually not, I've removed it.

**Review by dmitry-viskov — APPROVED** [review]
now look good to me! :+1:

File: edx/analytics/tasks/student_properties_per_ora_tags_dist.py (hunk 1 above)
Comment (dmitry-viskov) [reply edx/analytics/tasks/student_properties_per_ora_tags_dist.py]: We've added transformation `terms`, `runs` and `run` to `term`

**Pull request merged**